File: tests/fixtures/cpp/classes.cpp
namespace myapp {
    class Calculator {
    public:
        int add(int a, int b);
        void print(const std::string& msg);
    private:
        int value;
    };

    struct Point {
        double x;
        double y;
    };

    enum Color {
        RED,
        GREEN,
        BLUE
    };
}

File: tests/fixtures/csharp/classes.cs
namespace MyApp
{
    public class Calculator
    {
        public int Add(int a, int b)
        {
            return a + b;
        }

        public void Print(string message)
        {
            System.Console.WriteLine(message);
        }
    }

    public struct Point
    {
        public double X;
        public double Y;
    }

    public interface IProcessor
    {
        void Process(string data);
    }

    public enum Color
    {
        Red,
        Green,
        Blue
    }
}
